src/components/Card/Card.module.css

```diff
@@ -0,0 +1,54 @@
+.card {
+  --card-width: 250px;
```

Comment:
Is this the same value used in `PlayField` `grid-template-columns: repeat(4, 250px);`? If so, then it's better to define and use it there - it will also be available here.